packages/react-core/src/evm/providers/thirdweb-sdk-provider.tsx

```diff
@@ -124,7 +124,7 @@ const WrappedThirdwebSDKProvider = <
     type ForcedChainType = {
       rpc: string[];
       chainId: number;
-      nativeCurrency: { symbol: string; name: string; decimals: number };
+      nativeCurrency: { symbol: string; name: string; decimals: 18 };
```

Comment:
nah keep it number, there are chains out there that we support that have 16 decimals

Reply:
done